### PR TITLE
Adding Strip Hints

### PR DESCRIPTION
**Pull request by tanguycdls**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by radcheb** [comment]
👍

**Comment by tanguycdls** [comment]
@conda-forge-admin please re render ci

File: recipes/strip-hints/meta.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@
 # Jinja variables help maintain the recipe as you'll update the version only here.
 # Using the name variable with the URL in line 14 is convenient
 # when copying and pasting from another recipe, but not really needed.
-{% set name = "simplejson" %}
-{% set version = "3.8.2" %}
+{% set name = "strip-hints" %}
```
Comment (scopatz) [inline recipes/strip-hints/meta.yaml]: Please do not delete the example recipe

Comment (tanguycdls) [reply recipes/strip-hints/meta.yaml]: fixed

File: recipes/strip-hints/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
```
Comment (scopatz) [inline recipes/strip-hints/meta.yaml]: Please remove comments

File: recipes/strip-hints/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    from Python code files. The stripping process leaves runnable code, assuming the rest of the code is runnable
+     in the interpreter version. The program tries to make as few changes as possible to the processed code so that
+      line and column numbers in error messages for the processed code file also correspond to those for
+      the original code file. In most cases, with the default options, both the line and column numbers are preserved.
+  dev_url: https://github.com/abarker/strip-hints
+
+extra:
+  recipe-maintainers:
+    - tanguycdls
+    - radcheb
```
Comment (scopatz) [inline recipes/strip-hints/meta.yaml]: @radcheb are you willing to be a maintainer?

Comment (radcheb) [reply recipes/strip-hints/meta.yaml]: Ok to become a maintainer of `strip-hints` conda package.

File: recipes/strip-hints/meta.yaml (hunk 2 above)
Comment (tanguycdls) [reply recipes/strip-hints/meta.yaml]: yep fixed

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**